External/SPMmodified/xASL/xASL_bids_csv2tsvReadWrite.m

```diff
@@ -1,7 +1,7 @@
-function [PathTSV, CellContents] = xASL_bids_csv2tsvReadWrite(PathIn, bDeleteCSV, bWriteTSV)
+function [PathTSV, CellContents] = xASL_bids_csv2tsvReadWrite(PathIn, bDeleteCSV, bOverwrite)
```

Comment:
All changes of `bWriteTSV` to `bOverwrite` are illegal here. You can add `bOverwrite`, but you cannot remove `bWriteTSV`.

If `bWriteTSV` is false, and you read a CSV, it reads the content and puts it in memory. If `bWriteTSV` is true, it also reads to memory but additionally converts to TSV.

Reply:
Not entirely true - the code previously had bOverwrite in the code description, so I was only cleaning code. Not adding anything. So up to you to decide what names you want. I was just syncing it...

Reply:
The functionality is very different. `bOverwrite` means just overwriting yes or no. `bWriteTSV` means if you should convert a CSV-file to a TSV file (or only output the read CSV-file to memory). So this needs reverting.

Reply:
Agree. I was confused and tried to clean it. But I agree with your change.